tensorpack/decomposition.py

```diff
@@ -34,7 +34,9 @@ def __init__(self, data, matrix=[0], max_rr=5, method=perform_CP):
                 self.hasMatrix = True
         pass
 
-    def perform_tfac(self):
+    def perform_tfac(self, callback=None, rr_callback=1):
```

Comment:
These arguments should be described in a docstring.

Reply:
Adding now